src/components/victory-scatter.jsx

```diff
@@ -2,110 +2,236 @@ import React from "react";
-      case "star":
-        return "M0,3.5 L4.114,5.663 L3.329,1.082 L6.657,-2.163 L2.057,-2.832 L0,-7 " +
-          "L-2.057,-2.832 L-6.657,-2.163 L-3.329,1.082 L-4.114,5.663 L0,3.5";
+        return this._circlePath(x, y, size);
       case "square":
-        return "M4.5,-4.5 l0,9 l-9,0 l0,-9 l9,0";
+        return this._squarePath(x, y, size);
+      case "diamond":
+        return this._diamondPath(x, y, size);
       case "triangleDown":
-        return "M5.5,-4.763 l-5.5,9.526 l-5.5,-9.526 l11,0";
+        return this._triangleDownPath(x, y, size);
       case "triangleUp":
-        return "M5.5,4.763 l-5.5,-9.526 l-5.5,9.526 l11,0";
-      // no default
+        return this._triangleUpPath(x, y, size);
+      case "plus":
+        return this._plusPath(x, y, size);
+      case "star":
+        return this._starPath(x, y, size);
     }
   }
 
-  getSymbolPaths() {
-    return {
-      circle: this.getSymbolPath("circle"),
-      diamond: this.getSymbolPath("diamond"),
-      plus: this.getSymbolPath("plus"),
-      star: this.getSymbolPath("star"),
-      square: this.getSymbolPath("square"),
-      triangleDown: this.getSymbolPath("triangleDown"),
-      triangleUp: this.getSymbolPath("triangleUp")
-    };
+  _circlePath(x, y, size) {
+    return "M " + x + "," + y + " " +
+      "m " + -size + ", 0 " +
+      "a " + size + "," + size + " 0 1,0 " + (size * 2) + ",0 " +
+      "a " + size + "," + size + " 0 1,0 " + (-size * 2) + ",0";
   }
 
-  getScale(axis) {
-    let domain = this.props.domain;
-    const isXAxisData = axis === "x";
-    const range = isXAxisData ?
-      [this.props.maxBubbleRadius, this.props.width - this.props.maxBubbleRadius] :
-      [this.props.height - this.props.maxBubbleRadius, this.props.maxBubbleRadius];
-    const scale = d3.scale.linear().range(range);
+  _squarePath(x, y, size) {
+    return "M " + (x - size) + "," + (y + size) + " " +
+      "L " + (x + size) + "," + (y + size) +
+      "L " + (x + size) + "," + (y - size) +
+      "L " + (x - size) + "," + (y - size) +
+      "z";
+  }
 
-    if (_.isArray(domain)) {
-      return scale.domain(isXAxisData ? domain : domain.reverse());
-    } else if (domain && domain[axis]) {
-      return scale.domain(isXAxisData ? domain[axis] : domain[axis].reverse());
-    }
+  _diamondPath(x, y, size) {
+    const length = Math.sqrt(2 * (size * size));
+    return "M " + x + "," + (y + length) + " " +
+      "L " + (x + length) + "," + y +
+      "L " + x + "," + (y - length) +
+      "L " + (x - length) + "," + y +
+      "z";
+  }
 
-    domain = [_.min(_.pluck(this.props.data, axis)), _.max(_.pluck(this.props.data, axis))];
-    return scale.domain(isXAxisData ? domain : domain.reverse());
+  _triangleDownPath(x, y, size) {
+    const height = (size / 2 * Math.sqrt(3));
+    return "M " + (x - size) + "," + (y - size) + " " +
+      "L " + (x + size) + "," + (y - size) +
+      "L " + x + "," + (y + height) +
+      "z";
   }
 
-  getBubbleScale() {
```

Comment:
we need to scale circles (for bubble chart) by area, not by diameter. i used step 3 from this guide to make getBubbleScale: http://flowingdata.com/2010/11/23/how-to-make-bubble-charts/